postgres/changelog.d/17144.added

```diff
@@ -0,0 +1 @@
+Add support for collecting wal_bytes , wal_records, and wal_fpi for PG versions >= 13
```

Comment:
I would add the new metric names we are emitting & include that they are query metrics which required DBM to be enabled for collection

Reply:
not really, its just public facing documentation so deserves a little more scrutiny